postcss.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
I don't think we need a config file for postcss if it is just empty.

Reply:
Was facing this [No PostCSS Config found](https://github.com/akveo/ngx-admin/issues/604) issue. Solution seemed to be adding an empty postcss.config.js. I will look into if maybe i can resolve this some other way.

Reply:
Added (empty) options inline in js (at /storybook/lib/core/src/server/config/defaults/webpack.config.js) where the postcss-loader is being applied.